src/data/nodeid.cc

```diff
@@ -14,21 +14,61 @@ const uint8_t NodeID::ROOT_VALUE[NodeID::WIDTH_] = {
 };
 
 NodeID::NodeID() {
-  memcpy(value, NIL_VALUE, WIDTH_);
+  for (size_t i = 0; i < WIDTH_; i+=sizeof(unsigned int)) {
+    *reinterpret_cast<unsigned int*>(&value[i]) = random_();
```

Comment:
Fix this.
`random_` will be mersenne-twister, so I guess we can fill it byte-by-byte with decent performance.